src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -56,90 +57,107 @@ const ActiveState = ({
-                                onDelete={onCommentDelete}
-                                onEdit={onCommentEdit}
-                                permissions={{
-                                    can_delete: getProp(permissions, 'can_delete', false),
-                                    can_edit: getProp(permissions, 'can_edit', false),
-                                }}
-                                translations={translations}
-                            />
-                        </li>
-                    );
-                case 'task':
-                    return (
-                        <li
-                            key={type + id}
-                            className={classNames('bcs-activity-feed-task-new', { 'bcs-is-focused': isFocused })}
-                            data-testid="task"
-                            ref={refValue}
-                        >
-                            <TaskNew
-                                {...item}
-                                approverSelectorContacts={approverSelectorContacts}
-                                currentUser={currentUser}
-                                getApproverWithQuery={getApproverWithQuery}
-                                getAvatarUrl={getAvatarUrl}
-                                getUserProfileUrl={getUserProfileUrl}
-                                onAssignmentUpdate={onTaskAssignmentUpdate}
-                                onDelete={onTaskDelete}
-                                onEdit={onTaskEdit}
-                                onModalClose={onTaskModalClose}
-                                translations={translations}
-                            />
-                        </li>
-                    );
-                case 'file_version':
-                    return (
-                        <li key={type + id} className="bcs-version-item" data-testid="version">
-                            {versions ? (
-                                <CollapsedVersion {...item} onInfo={onVersionInfo} />
-                            ) : (
-                                <Version {...item} onInfo={onVersionInfo} />
-                            )}
-                        </li>
-                    );
-                case 'keywords':
-                    return (
-                        <li key={type + id} className="bcs-keywords-item" data-testid="keyword">
-                            <Keywords {...item} />
-                        </li>
-                    );
-                case 'app_activity':
-                    return (
-                        <li key={type + id} className="bcs-activity-feed-app-activity" data-testid="app-activity">
-                            <AppActivity currentUser={currentUser} onDelete={onAppActivityDelete} {...item} />
-                        </li>
-                    );
-                default:
-                    return null;
-            }
-        })}
-    </ul>
-);
+    const isValidType = feedEntryType => {
```

Comment:
Could do implicit return here. Also can we use the existing constants for 'task' and 'comment'?